server/tests/common/data.py

```diff
@@ -211,3 +215,167 @@ async def _alice_generated_realm_data(now: DateTime) -> None:
+                    now=DateTime.now(),
+                    organization_id=coolorg.organization_id,
+                    author=coolorg.bob.device_id,
+                    author_verify_key=coolorg.bob.signing_key.verify_key,
+                    revoked_user_certificate=certif.dump_and_sign(coolorg.alice.signing_key),
+                )
+                assert isinstance(outcome, RevokedUserCertificate)
+
+                expected_http_status = 461
+
+            case "author_frozen":
+                outcome = await backend.user.freeze_user(
+                    organization_id=coolorg.organization_id,
+                    user_id=coolorg.alice.user_id,
+                    user_email=None,
+                    frozen=True,
+                )
+                assert isinstance(outcome, UserInfo)
+
+                expected_http_status = 462
+
+            case unknown:
+                assert False, unknown
+
+        try:
+            await do()
+            assert False, f"{do!r} was expected to raise an `RpcTransportError` exception !"
+        except RpcTransportError as err:
+            assert err.rep.status_code == expected_http_status
+
+    yield _http_46x_authenticated_tester
+
+    assert tester_called
+
+
+# TODO: not sure how to test "organization_not_found" & "invitation_not_found"
+@pytest.fixture(params=("organization_expired", "invitation_already_used"))
+async def http_46x_invited_tester(
+    request: pytest.FixtureRequest, coolorg: CoolorgRpcClients, backend: Backend
+) -> AsyncGenerator[Http46xTester, None]:
+    tester_called = False
+
+    async def _http_46x_invited_tester(do: Http46xTesterDoCallback):
+        nonlocal tester_called
+        tester_called = True
+
+        match request.param:
+            case "organization_expired":
+                outcome = await backend.organization.update(
+                    id=coolorg.organization_id, is_expired=True
+                )
+                assert outcome is None
+
+                expected_http_status = 460
+
+            case "invitation_already_used":
+                outcome = await backend.invite.cancel(
+                    now=DateTime.now(),
+                    organization_id=coolorg.organization_id,
+                    author=coolorg.alice.device_id,
+                    token=coolorg.invited_alice_dev3.token,
+                )
+                assert outcome is None
+
+                expected_http_status = 410
+
+            case unknown:
+                assert False, unknown
+
+        try:
+            await do()
+            assert False, f"{do!r} was expected to raise an `RpcTransportError` exception !"
+        except RpcTransportError as err:
+            assert err.rep.status_code == expected_http_status, err
+
+    yield _http_46x_invited_tester
+
+    assert tester_called
+
+
+# TODO: not sure how to test "organization_not_found"
+@pytest.fixture(params=("organization_expired",))
+async def http_46x_anonymous_tester(
```

Comment:
I'd rename this functions (See comment below in `test_api.py`). Also I would put family first, I think is clearer when using it as fixture.

Something like:
```suggestion
async def anonymous_http_common_errors_tester(
```